src/App/MineAnsatte/SideBytter/Pagingknapper/PagineringsKnapp/PagineringsKnapp.less

```diff
@@ -1,36 +1,37 @@
 @import '../../../../../../node_modules/nav-frontend-core/less/_variabler';
 
-.valg {
-    -webkit-border-radius: 27px;
-    -moz-border-radius: 27px;
-    border-radius: 27px;
-    border-color: transparent;
-    width: 25px;
-    height: 25px;
-    padding: 1px 2px 0 2px;
-    text-align: center;
-    background-color: @navLysGra;
-    color: black;
-    align-self: center;
-    margin: 0 0.5rem 0 0.5rem;
-
-    &:focus {
-        z-index: 16;
-        background-color: @navBlaLighten80;
-        outline: none;
-        box-shadow: 0 0 0 3px @fokusFarge;
-    }
+.sidebytter {
+    &__valg {
+        -webkit-border-radius: 27px;
+        -moz-border-radius: 27px;
+        border-radius: 27px;
+        border-color: transparent;
+        width: 25px;
+        height: 25px;
+        padding: 1px 2px 0 2px;
+        text-align: center;
+        background-color: @navLysGra;
+        color: black;
+        align-self: center;
+        margin: 0 0.5rem 0 0.5rem;
 
-    &__sidetall {
-        font-size: 14px;
+        &:focus {
+            background-color: @navBlaLighten80;
+            outline: none;
+            box-shadow: 0 0 0 3px @fokusFarge;
+            color: black;
+        }
+
+        &__sidetall {
+            font-size: 14px;
```

Comment:
Vi burde kanskje bruke 0.875rem her, sånn at de som har behov for å zoome inn på teksten kan gjøre det? :D